models/dataerror.py

```diff
@@ -0,0 +1,3 @@
+class DataValidationError(Exception):
```

Comment:
I would probably store this in the root directory. Not the `models` directory.

Reply:
I followed the code from the proffesor. He had it inside the models.py file.

Reply:
I think this is fine!

Reply:
Well, I think it was so his code was simpler to follow. Since we're organizing models in it's own directory and DataValidationError isn't a model, I personally don't think it makes sense to keep it there.

If you look at one of his later projects, you'll see some differences: https://github.com/nyu-devops/lab-flask-bdd/tree/master/app.

But again, this is an MVP so if I'm outnumbered then so be it. 😄